test/test_playbooks/organization.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: localhost
+  tasks:
+  - name: Load server config
+    include_vars:
+      file: server_vars.yml
+  - include_tasks: tasks/organization.yml
```

Comment:
I just figured out, `include_tasks` is only available since ansible 2.4.
https://docs.ansible.com/ansible/2.4/include_tasks_module.html
Do we want to be able to run the tests on 2.3? (At least that's what our modules describe as a dependency.)

Reply:
Yea, i have seen the warning too. It would be nicer, if they warned one version later. This means you cannot write playbooks without incompatibility and warning for two consecutive minor versions of ansible. 😞